README.md

```diff
@@ -719,6 +719,7 @@ Table of Contents
   * [Adobe XD](https://www.adobe.com/products/xd.html) - Wireframe & Prototyping tool similar to Sketch. Free plan covers: 1 active shared design spec, Adobe Fonts Free (limited set of fonts), 2GB of cloud storage.
   * [AllTheFreeStock](https://allthefreestock.com) - a curated list of free stock images, audio and videos.
   * [CodeMyUI](https://codemyui.com) - Handpicked collection of Web Design & UI Inspiration with Code Snippets.
+  * [Pencil](https://github.com/evolus/pencil) - Open source design tool using Electron. 
```

Comment:
thanks for the contribution, generally this list is a list solely for SaaS tools and pencil appears to be a tool you download but we make exceptions for IDEs so this seems close enough to that.

thanks!